blazor/datagrid/editing.md

```diff
@@ -1279,6 +1279,53 @@ In the following sample, the `SfCheckbox` component is rendered as a template in
 
 > [View Sample in GitHub.](https://github.com/SyncfusionExamples/blazor-datagrid-single-click-editing-with-boolean-column)
 
+## How to make a Grid column always editable
+
+You can make the Grid column always editable by using the [column template](https://blazor.syncfusion.com/documentation/datagrid/column-template) feature of the Grid.
```

Comment:
You can make the Grid column always editable using the [column template](https://blazor.syncfusion.com/documentation/datagrid/column-template) feature of the Grid.